core/src/main/java/io/aiven/klaw/service/ClusterApiService.java

```diff
@@ -1126,6 +1127,55 @@ public Map<String, String> retrieveMetrics(String jmxUrl, String objectName)
     }
   }
 
+  public ApiResponse resetConsumerOffsets(
+      ResetConsumerGroupOffsetsRequest resetConsumerGroupOffsetsRequest,
+      String environmentId,
+      int tenantId)
+      throws KlawException {
+    log.info("ResetConsumerGroupOffsetsRequest {}", resetConsumerGroupOffsetsRequest);
+    getClusterApiProperties(tenantId);
+    try {
+      Env envSelected = manageDatabase.getHandleDbRequests().getEnvDetails(environmentId, tenantId);
+      ResponseEntity<ApiResponse> response;
+
+      KwClusters kwClusters =
+          manageDatabase
+              .getClusters(KafkaClustersType.KAFKA, tenantId)
+              .get(envSelected.getClusterId());
+
+      String uriResetConsumerOffsets =
+          clusterConnUrl
+              + URI_CONSUMER_OFFSETS_RESET
+              + kwClusters.getBootstrapServers()
+              + "/"
+              + kwClusters.getProtocol()
+              + "/"
+              + kwClusters.getClusterName()
+              + kwClusters.getClusterId();
+
+      HttpHeaders headers = createHeaders(clusterApiUser);
+      HttpEntity<ResetConsumerGroupOffsetsRequest> request =
+          new HttpEntity<>(resetConsumerGroupOffsetsRequest, headers);
+      response =
+          getRestTemplate()
+              .exchange(
+                  uriResetConsumerOffsets,
+                  HttpMethod.POST,
+                  request,
+                  new ParameterizedTypeReference<>() {});
+
+      return response.getBody();
+
+    } catch (Exception e) {
+      log.error("Error from resetConsumerOffsets ", e);
+      if (e.getMessage().contains(CLUSTER_API_ERR_120)
```

Comment:
are we sure that here `e` always has a message? I'm asking since otherwise there could be `NullPointerException` on this line